viaacode/hetarchief-client
# Feature/upgrade admin core UI

Upgrade the `admin-core-ui` package to avoid legacy code.

To test this code you'll need to:
- check out the `feature/admin-core-reduce-duplicate-code` branch for the proxy
- update the `.env` file with the latest version which is stored in 1PW
